src/components/interaction-designer/block-types/MobilePrimitives_OpenResponseBlock.vue

```diff
@@ -13,6 +13,8 @@
         :block="block" />
       <block-name-editor :block="block" />
 
+      <hr>
```

Comment:
I don't think we need `<hr>` before `block-max-duration-seconds-editor`, right ?

Reply:
<img width="405" alt="Screen Shot 2021-07-26 at 4 20 08 PM" src="https://user-images.githubusercontent.com/23651142/127071323-185f3fbd-57ad-4f61-aa7d-54f1e6a2e83b.png">

@safydy-r  As seen in this Design screenshot (DSN-111). It has a divider between the title and rest of the block fields.

Reply:
I see, you're right. I might have been confused somewhere :D